pkgs/development/libraries/polkit/default.nix

```diff
@@ -55,7 +55,8 @@ stdenv.mkDerivation rec {
     ++ [ libxslt docbook_xsl docbook_xml_dtd_412 ]; # man pages
   buildInputs =
     [ expat pam spidermonkey_60 ]
-    ++ (if useSystemd then [systemd] else [elogind])
+    # On Linux, fall back to elogind when systemd support is of.
+    ++ (if useSystemd then [systemd] else stdenv.lib.optional stdenv.isLinux [elogind])
```

Comment:
```suggestion
    ++ (if useSystemd then [systemd] else stdenv.lib.optional stdenv.isLinux elogind)
```

Reply:
Ah right. I've slightly rewritten it, pulling the Linux condition into the front, which I think is clearer.